doc/examples/applications/plot_3d_segmentation_embryo.py

```diff
@@ -0,0 +1,206 @@
+"""
+==================================================
+Segment 3D image sample of developing mouse embryo
+==================================================
+
+In this example, we look at a microscopy image of a developing mouse embryo.
+We use sample data from [1]_, more precisely from embryo B at time point 184.
+
+.. [1] McDole K, Guignard L, Amat F, Berger A, Malandain G, Royer LA,
+       Turaga SC, Branson K, Keller PJ (2018) "In Toto Imaging and
+       Reconstruction of Post-Implantation Mouse Development at the
+       Single-Cell Level" Cell, 175(3):859-876.e33.
+       ISSN: 0092-8674
+       :DOI:`10.1016/j.cell.2018.09.031`
+
+"""
+
+import io
+import requests
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import ndimage as ndi
+
+import skimage as ski
+
+
+#####################################################################
+# We downloaded the original data in KLB format and sliced a particular
+# subvolume, which we saved into a compressed Numpy format:
+#
+# .. code-block:: python
+#
+#     import numpy as np
+#     import pyklb as klb
+#
+#     data = klb.readfull('Mmu_E1_CAGTAG1.TM000184_timeFused_blending/SPM00_TM000184_CM00_CM01_CHN00.fusedStack.corrected.shifted.klb')
+#     sample = data[400:450, 1000:1750, 400:900]
+#     np.savez_compressed('sample_3D_frame_184.npz', sample)
```

Comment:
I was originally looking at the rendered example and was wondering / confused on why this code was passing even though `pyklb` isn't available when building the docs.

This might confuse readers in a similar manner; is this really relevant here? Or how we can make it more clear that this is not part of the examples workflow?

Reply:
The rendering of such code blocks used to make it clear that they weren't part of the current workflow... They would appear as 'commented out,' sort of. I had used this formatting in a previous example ("Estimate anisotropy in a 3D microscopy image"). I can see that it's not the case anymore (with the new theme, I guess).

Yes, it would be best to find some alternative formatting...

Reply:
Maybe a codeblock within a citation block? :thinking:

Reply:
> Maybe a code block within a citation block? 🤔

Good idea, @lagru! I also added a caption:
![code_in_quote](https://github.com/scikit-image/scikit-image/assets/2227806/3f3431bd-8748-4531-b37f-25b87ac479ea)
What do you think?

Reply:
Applied in all three places at 1092745319c48.

Reply:
The quoted code block looks perfect! Especially with the title functionality I didn't know about. :)